src/course-unit/move-modal/components/EmptyMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from '../messages';
+
+const EmptyMessage = ({ category, categoryText }: { category: string, categoryText: string }) => {
+  const intl = useIntl();
+  return (
+    <li className="xblock-no-child-message">
+      {intl.formatMessage(messages.moveModalEmptyCategoryText, {
+        category,
+        categoryText: categoryText.toLowerCase(),
+      })}
+    </li>
+  );
+};
+
+EmptyMessage.propTypes = {
+  category: PropTypes.string.isRequired,
+  categoryText: PropTypes.string.isRequired,
+};
```

Comment:
You can omit `propTypes` from `.tsx` files if you want.

Reply:
Removed